lib/index.coffee

```diff
@@ -10,8 +10,8 @@ dockerUtils = require './docker'
 class DockerGC
 	setDocker: (hostObj) ->
 		@currentMtimes = {}
-		@hostObj = hostObj
-		dockerUtils.getDocker(hostObj)
+		@hostObj = _.defaults({ Promise }, hostObj)
```

Comment:
This will always use bluebird for the `Promise` option - is that what you intend?

Reply:
Given that this is internal to the module, I think this is fine.

Reply:
@Page- yes that is the idea.
We have code that calls the Dockerode object methods and the runs `.map`s on the returned promises.